tests/HUBViewControllerTests.m

```diff
@@ -91,7 +91,7 @@ @interface HUBViewControllerTests : XCTestCase <HUBViewControllerDelegate>
 @property (nonatomic, assign) BOOL didReceiveViewControllerDidFinishRendering;
 @property (nonatomic, copy) void (^viewControllerDidFinishRenderingBlock)(void);
 @property (nonatomic, copy) BOOL (^viewControllerShouldStartScrollingBlock)(void);
-@property (nonatomic, copy) BOOL (^viewControllerShouldIgnoreHeaderComponentInset)(void);
+@property (nonatomic, copy) BOOL (^viewControllerShouldAutomaticallyManageTopContentInset)(void);
```

Comment:
I think we started doing it because we needed flexibility of controlling the implementation in some places, and then as multiple devs have added new tests they've stuck to that convention. I'm fine with changing these to simple `BOOL`s, but it should be done in a separate PR that doesn't contain any logic changes, for satefy 😄